### PR TITLE
chore: add pr template

### PR DESCRIPTION
**Pull request by greatng**
This should come in handy when there are more contributors.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Issue ticket number and link
+
+<https://github.com/wevisdemo/parliament-watch/issues/YOUR_ISSUE_NUMBER>
```
Comment (Th1nkK1D) [inline .github/pull_request_template.md:3]: GitHub has an official way to link issues to the PR. And if PR is merged to `main` branch, the issue will be closed automatically. [See more detail here.](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

We want contributors to use this method. So my suggestion is

```md
## Related GitHub issues

Close #<tag-related-issue-number-here>
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+## What have been done
+
+- Describe what you have done
+-
+
+## Screenshot (if any)
```
Comment (Th1nkK1D) [inline .github/pull_request_template.md:10]: Suggesting adding more section: "Help needed (if any)"

**Review by Th1nkK1D — COMMENTED** [review]
This is nice and surely will help future contributors. I do have a few suggestions.

File: .github/pull_request_template.md (hunk 1 above)
Comment (greatng) [reply .github/pull_request_template.md]: Done, thanks for document.

File: .github/pull_request_template.md (hunk 2 above)
Comment (greatng) [reply .github/pull_request_template.md:10]: Added krub

**Pull request merged**